### ISSUE TITLE
onevm: disk-snapshot-create does not handle error if live snapshot feature is disabled

### ISSUE THREAD

**Issue by abbbi** [OpenNebula/one#5943]
**Description**

If the live snapshot feature is disabled, onevm disk-snapshot-create still exits with zero exit code and reports that an
snapshot was successfully created.

**To Reproduce**

1)  in /etc/one/vmm_exec/vmm_execrc set LIVE_DISK_SNAPSHOTS=""
2) Restart services (not sure if required)
3) Attempt to create an snapshot:

root@debian11-one:/home/vagrant# onevm disk-snapshot-create 1 0 TESTSNAPSHOT2 -v
VM 1: disk snapshot created
root@debian11-one:/home/vagrant# echo $?
0

4) in the VM logfile it is to be seen, that snapshot creation failed:

```
Wed Aug 10 10:13:07 2022 [Z0][VM][I]: New state is ACTIVE
Wed Aug 10 10:13:07 2022 [Z0][VM][I]: New LCM state is DISK_SNAPSHOT
Wed Aug 10 10:13:07 2022 [Z0][VMM][E]: DISKSNAPSHOTCREATE: Cannot perform a live DISKSNAPSHOTCREATE operation

```

**Expected behavior**

onevm disk-snapshot-create should exit with error code != 0 and report propper error.

**Details**
Version used: opennebula 6.2 on Debian Bullseye.
 
## Progress Status
- [ ] Code committed
- [ ] Testing - QA
- [ ] Documentation (Release notes - resolved issues, compatibility, known issues)

**Comment by paczerny** [comment]
The CLI command `onevm disk-snapshot-create` sends a xml-rpc request to `oned`. The oned checks if the action can be executed (VM in correct state, disk exists, ...). If all conditions pass, it calls asynchronous driver action and the CLI reports success.

If there is an error in the asynchronous action it is loged in the VM->USER_TEMPLATE->ERROR, in the  `/var/log/one/xx.log` and in the `/var/log/one/oned.log`. The settings in `vmm_execrc` is one of these errors.

It's not possible to report this error as return code of the CLI action.

**Comment by atodorov-storpool** [comment]
In the case of asynchronous calls, what do you think about the driver/cli to return 

> disk snapshot **request accepted** (or _pending_?)

and clarify in the documentation that this is an asynchronous call and it is important to monitor the VM status for further information?

**Comment by rsmontero** [comment]
@atodorov-storpool Agree, CLI output is misleading `VM 1: disk snapshot created` should be replaced by "being created" or similar. Also I think it is a good idea to flag API calls that are "instant", as opposed to those that require a driver action and progress needs to be checked through state changes, or VM information....
